src/message/Message.js

```diff
@@ -1,13 +1,27 @@
 import React, { PropTypes } from 'react';
 
-const Message = ({ children, params, className }, { translations: { translate } }) => (
-  <span className={className}>{translate(children, params)}</span>
-);
+const Message = (
```

Comment:
I think this part is complicated for future maintainers (possibly?). Destructuring and arrow functions all at once can be confusing for people less familiar. But it's not wrong by any means.